### PR TITLE
Wrote release notes for Catalog 2.95.0 release

### PR DESCRIPTION
**Pull request by NammNguyen**

### CODE REVIEW

File: content/en/docs/releasenotes/catalog/_index.md [hunk 1]
```diff
@@ -17,11 +17,11 @@ These release notes cover changes made to the [Catalog](/catalog/).
 
 ### Features
 
-* We added a mini-survey on the catalog usefulness to search results page. Let us know how we're doing!
+* We added a mini survey about the Catalog's usefulness to the Search Results page so you can let us know how we're doing.
```
Comment (Adam-Dupaski) [inline content/en/docs/releasenotes/catalog/_index.md]: Does "Search Results" appear in the UI capitalized, and if so, should it be bolded here?

"we're" > "we are" (we do not use contractions)

File: content/en/docs/releasenotes/catalog/_index.md [hunk 2]
```diff
@@ -17,11 +17,11 @@ These release notes cover changes made to the [Catalog](/catalog/).
...
 
 ### Improvement
 
-* We updated the link on the search results page to explain more clearly what OpenAPI beta means in the Catalog
+* We updated the link on the Search Results page to give more details about what OpenAPI Beta means in the Catalog.
```
Comment (Adam-Dupaski) [inline content/en/docs/releasenotes/catalog/_index.md]: Would be helpful to add a cross-reference here

File: content/en/docs/releasenotes/catalog/_index.md [hunk 3]
```diff
@@ -17,11 +17,11 @@ These release notes cover changes made to the [Catalog](/catalog/).
```
Comment (Adam-Dupaski) [inline content/en/docs/releasenotes/catalog/_index.md:17]: Something to change on the date?

File: layouts/partials/landingpage/latest-releases.html [hunk 4]
```diff
@@ -16,7 +16,7 @@
                                                 <p class="rn-date">Sep 19, 2023</p>
 
 	<li class="link-list"><a href="/releasenotes/data-hub/">Catalog</a></li>
-												<p class="rn-date">Sep 21, 2023</p>
+												<p class="rn-date">Oct 05, 2023</p>
```
Comment (Adam-Dupaski) [inline layouts/partials/landingpage/latest-releases.html]: Something to change on the date?

File: content/en/docs/releasenotes/catalog/_index.md (hunk 2 above)
Comment (quinntracy) [reply content/en/docs/releasenotes/catalog/_index.md]: I reached out to Nam for the survey link, so I'll add it in once he responds

Comment (Adam-Dupaski) [reply content/en/docs/releasenotes/catalog/_index.md]: Survey link should go in first release note.

In this release note, cross-referencing the doc for "OpenAPI Beta" would be good.

File: layouts/partials/landingpage/latest-releases.html (hunk 4 above)
Comment (NammNguyen) [reply layouts/partials/landingpage/latest-releases.html]: What do you mean here? We are going to release it tomorrow

File: content/en/docs/releasenotes/catalog/_index.md (hunk 2 above)
Comment (NammNguyen) [reply content/en/docs/releasenotes/catalog/_index.md]: It's an in-app survey that shows up once the user performs a certain action, no link required.

File: content/en/docs/releasenotes/catalog/_index.md (hunk 1 above)
Comment (NammNguyen) [reply content/en/docs/releasenotes/catalog/_index.md]: The text "Search Results" does not appear in the UI at all

File: content/en/docs/releasenotes/catalog/_index.md (hunk 2 above)
Comment (NammNguyen) [reply content/en/docs/releasenotes/catalog/_index.md]: https://docs.mendix.com/catalog/register/openapi-automatic-registration/ 
Could be used for cross reference

Comment (quinntracy) [reply content/en/docs/releasenotes/catalog/_index.md]: Thanks, Nam!

File: content/en/docs/releasenotes/catalog/_index.md (hunk 1 above)
Comment (quinntracy) [reply content/en/docs/releasenotes/catalog/_index.md]: Thanks for clarifying this, Nam

File: layouts/partials/landingpage/latest-releases.html (hunk 4 above)
Comment (Adam-Dupaski) [reply layouts/partials/landingpage/latest-releases.html]: @quinntracy

**Pull request merged**